posts/2020-05-27-how-to-analyze-open-liberty-logs-with-splunk.adoc

```diff
@@ -0,0 +1,135 @@
+---
+layout: post
+title: "How to Analyze Open Liberty Logs with Splunk"
+categories: blog
+author_picture: https://avatars0.githubusercontent.com/u/29461649
+author_github: https://github.com/pgunapal
+seo-title: How to Analyze Open Liberty Logs with Splunk - OpenLiberty.io
+seo-description: When asked to identify which log analysis tool they use with Open Liberty, Splunk is one of the most common responses from our users.  Log analysis is undoubtedly a key part of monitoring your applications.  Collecting log data from Docker containers running Open Liberty in one place and monitoring, searching, and visualizing those logs in real-time results in reduced time, to find and troubleshoot problems – let’s take a look at analyzing logs from an Open Liberty Docker container, using Splunk. 
+blog_description: When asked to identify which log analysis tool they use with Open Liberty, Splunk is one of the most common responses from our users.  Log analysis is undoubtedly a key part of monitoring your applications.  Collecting log data from Docker containers running Open Liberty in one place and monitoring, searching, and visualizing those logs in real-time results in reduced time, to find and troubleshoot problems – let’s take a look at analyzing logs from an Open Liberty Docker container, using Splunk. 
+---
+= How to Analyze Open Liberty Logs with Splunk
+Prashanth Gunapalasingam <https://github.com/pgunapal>
+
+When we ask our users which log analysis tool they use with Open Liberty, one of the most common responses is Splunk. Collecting log data from running servers in one place and monitoring, searching, and visualizing those logs in real-time helps you find and troubleshoot problems faster. There are many tools available to aggregate and analyze the log events of your application, where each of them have their own benefits to particular use cases. Splunk provides a forwarder, which pushes data to remote indexers via HTTP/HTTPS, which allows easier and secure analysis of logs from running Docker containers. Let’s take a look at using Splunk to analyze logs from an Open Liberty Docker container.
```

Comment:
**change**
There are many tools available to aggregate and analyze the log events of your application, where each of them have their own benefits to particular use cases. Splunk provides a forwarder, which pushes data to remote indexers via HTTP/HTTPS, which allows easier and secure analysis of logs from running Docker containers.

**to**
There are many tools available to aggregate and analyze the log events of your application, and each of them have their own benefits for particular use cases. Splunk provides a forwarder that pushes data to remote indexers via HTTP/HTTPS, which enables easier secure analysis of logs from running Docker containers.